src/components/Icon/Icon.tsx

```diff
@@ -1,10 +1,14 @@
+import React from "react";
```

Comment:
looks like this component doesn't do anything but returning an img.
you can remove it and use normal img instead unless you are planning to do something with it.